platform/qt/src/sqlite3.cpp

```diff
@@ -60,20 +61,29 @@ void checkDatabaseOpenError(const QSqlDatabase &db) {
     }
 }
 
+namespace {
+    static QAtomicInt count = 0;
+
+    // Qt4 does not support operator++ in QAtomicInt.
+    QString incrementCount() {
+        count = count + 1;
```

Comment:
This is not exactly atomic, because you have in fact two operations.

I think you could do a `return QString::number(count.fetchAndAddAcquire(1));`